test/views/EORIBeUpToDateViewSpec.scala

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package views
+
+import models.AuthUserType.{IndividualTrader, OrganisationUser}
+import models.DraftId
+import play.twirl.api.HtmlFormat
+import views.behaviours.ViewBehaviours
+import views.html.EORIBeUpToDateView
+
+class EORIBeUpToDateViewSpec extends ViewBehaviours {
+
+  val messageKeyPrefix = "eoriBeUpToDate"
+
+  val view: EORIBeUpToDateView = app.injector.instanceOf[EORIBeUpToDateView]
+
+  val viewViaApply: HtmlFormat.Appendable  = view(DraftId(1L), IndividualTrader)(fakeRequest, messages)
+  val viewViaRender: HtmlFormat.Appendable = view.render(DraftId(1L), IndividualTrader, fakeRequest, messages)
+  val viewViaF: HtmlFormat.Appendable      = view.f(DraftId(1L), IndividualTrader)(fakeRequest, messages)
+
+  "EORIBeUpToDateView" - {
+    normalPage(messageKeyPrefix, "")()
+  }
+
+  "when not an Individual Trader" - {
+    val viewAlternate: HtmlFormat.Appendable = view(DraftId(1L), OrganisationUser)(fakeRequest, messages)
+
+    pageByMethod(viewAlternate, messageKeyPrefix, "org")()
+  }
```

Comment:
Not sure if you have moved it up yet but this should be within the `EORIBeUpToDateView` {}. Just wanted to post this so it is not missed, thanks